test/e2e/k8s_util.go

```diff
@@ -832,6 +840,35 @@ func (k *KubernetesUtils) GetCG(name string) (*crdv1beta1.ClusterGroup, error) {
 	return k.crdClient.CrdV1beta1().ClusterGroups().Get(context.TODO(), name, metav1.GetOptions{})
 }
 
+// CreateGroup is a convenience function for creating an Antrea Group by namespace, name and selector.
+func (k *KubernetesUtils) CreateGroup(namespace, name string, pSelector, nSelector *metav1.LabelSelector, ipBlocks []crdv1beta1.IPBlock) (*crdv1beta1.Group, error) {
+	log.Infof("Creating group %s/%s", namespace, name)
+	_, err := k.crdClient.CrdV1alpha3().Groups(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
```

Comment:
I had a PR to simplify such code: no need to get a resource first as create will fail with a specific AlreadyExists error anyway.